parking_permits_app/admin.py

```diff
@@ -72,7 +73,7 @@ class LowEmissionCriteriaAdmin(admin.ModelAdmin):
 
 
 @admin.register(ParkingPermit)
-class ParkingPermitAdmin(admin.ModelAdmin):
+class ParkingPermitAdmin(VersionAdmin):
```

Comment:
I actually deliberately leave this out when adding the reversion thing. IMO, we should not allow making versioned changes through Django admin for the parking permits because the `VersionAdmin` uses Django's `LogEntry` object to generate the change message, which will cause problems when parsing the changelogs for displaying in the admin UI.

Reply:
Ok. Removed the commit.